### PR TITLE
Calculator app: Milestone 2

### PR DESCRIPTION
**Pull request by Temmarie**
In this milestone, I added and implemented the following components:

- `App`
- `Button`
- `ButtonPanel`
- `Display`

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div id="App">
+      <Display />
+      <ButtonPanel />
+    </div>
```
Comment (whiz25) [inline src/components/App.js:10]: - Use React fragment to wrap the components
```javascript
<>
...
</>
```
Read more [here](https://reactjs.org/docs/fragments.html#short-syntax)

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Changes Required :pray:
Good job @Temmarie . Your PR meets most of the requirements for this milestone.
Check my comment for suggestion on how to fulfill the remaining requirement.

### Martin Afani

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @Temmarie 👋 
Great job implementing the changes as suggested. 👏 
Some areas need attention in the codebase. 👀 
Let's take care of them before moving forward, 👇 

## Changes 
- [ ] Remove the `src/App.js` and `src/App.test.js` files from the project directory as they are not required in this milestone. The App file has no purpose and the tests in the src are not testing anything, I'd suggest you remove them for now. thank you! 
### Happy coding 💻 
#### Christian Otieno (CR)

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

**Pull request merged**